src/groups/special_euclidean.jl

```diff
@@ -72,34 +83,38 @@ Base.@propagate_inbounds function Base.setindex!(
 end
 
 Base.@propagate_inbounds function submanifold_component(
-    ::Union{SpecialEuclidean{n},SpecialEuclideanManifold{n}},
```

Comment:
So in the old form we implemented it for every union of same size but actually that was ten then not necessary? Ah, we still need the `n` now, I see. But here, we would loose the “dispatch-advantage” of the type-manifold, right?

Reply:
What "dispatch-advantage” of the type-manifold" do you mean?

Reply:
Ah maybe not so relevant – the new form looks nice and seems to work for all ur diverse manifold types, that nice.